streamlit_folium/frontend/package.json

```diff
@@ -8,12 +8,11 @@
     "@testing-library/user-event": "^7.1.2",
     "@types/hoist-non-react-statics": "^3.3.1",
     "@types/jest": "^24.0.0",
-    "@types/leaflet": "^1.7.9",
     "@types/node": "^12.0.0",
     "bootstrap": "^4.4.1",
     "event-target-shim": "^5.0.1",
     "hoist-non-react-statics": "^3.3.2",
-    "leaflet": "^1.7.1",
+    "leaflet": "^1.9.4",
```

Comment:
@blackary or @hansthen...with the dynamic insertion of the JS libraries that we now do, do we even need to have leaflet and its peers as part of the dependencies?

Reply:
My guess is they can be safely removed. I did a quick test building without these dependencies, and running the tests. The `test_layer_control_dynamic_update` passed only sometimes :-(. I don't think that should be related to the dependencies though. If you want I can make a PR for the changes.

Reply:
No need to make a PR, just trying to lean on your conceptual knowledge of what comes out of Folium :)